Maui.FreakyEffects/Maui.FreakyEffects/Shared/Skeleton/SkeletonEffect.cs

```diff
@@ -187,7 +187,7 @@ private static void RestoreBackgroundColor(View view)
         }
         else if (backgroundColor != default(Color))
         {
-            view.BackgroundColor = GetOriginalBackgroundColor(view);
+            view.BackgroundColor = GetOriginalBackgroundColor(view) ?? Colors.Transparent;
```

Comment:
If OriginalBackgroundColor is null it will not change so setting to transparent instead seems to do the trick

Reply:
Interesting let me take a look at this